gcp_pilot/calendar.py

```diff
@@ -171,6 +171,7 @@ def create_or_update_event(
         recurrence_amount: str = None,
         calendar_id: str = "primary",
         color: Color = None,
+        send_notifications: str = None
```

Comment:
It's a boolean, right?
https://developers.google.com/calendar/api/v3/reference/events/insert